res/controllers/Denon-MC7000-scripts.js

```diff
@@ -653,19 +653,22 @@ MC7000.loadButton = function(channel, control, value, status, group) {
 MC7000.wheelTouch = function(channel, control, value, status, group) {
     var deckNumber = script.deckFromGroup(group);
     var deckOffset = deckNumber - 1;
-    if (MC7000.isVinylMode[deckOffset]) {
+    var maxLibrary = engine.getValue("[Master]", "maximize_library");
+    if (MC7000.isVinylMode[deckOffset] && maxLibrary === 0) {
```

Comment:
```suggestion
    var libraryMaximized = engine.getValue("[Master]", "maximize_library") > 0;
    if (MC7000.isVinylMode[deckOffset] && !libraryMaximized) {
```
This is a boolean conceptually a boolean so it makes sense to use it as one in the code.

Keep in mind there are other places where `maxLibrary` is used as well.

Reply:
done - still dont understand the difference. It is basically the same. For me this ranks again to personal taste change request.